pyprql/tests/test_magic.py

```diff
@@ -269,6 +269,22 @@ def test_autopandas(ip):
     assert dframe.name[0] == "foo"
 
 
+def test_target_dialect(ip):
+    ip.run_line_magic("config", 'PrqlMagic.target = "sql.sqlite"')
+    dframe = run_prql(
+        ip, 'from author | select foo = f"{first_name}-{last_name}" | take 1'
+    )
+    assert dframe.foo[0] == "William-Shakespeare"
+
+
+@pytest.mark.xfail(reason="no such function: CONCAT")
```

Comment:
`xfail` is generally for things that we would like to pass but don't. Here, do we want to use `raises`? (i.e. I think we _want_ this to fail?)